src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1717,6 +1718,38 @@ public String getVariableNamePattern() {
 					})
 					.serializer(new EnumSerializer<>(Cat.Type.class)));
 		}
+		
+		EnumUtils<Attribute> attributes = new EnumUtils<>(Attribute.class, "attribute types");
+		Classes.registerClass(new ClassInfo<>(Attribute.class, "attributetype")
+				.user("attribute ?types?")
+				.name("Attribute Type")
+				.description("The attribute type of an entity. Note that this type does not contain any numerical values."
+						+ "See <a href='https://minecraft.gamepedia.com/Attribute#Attributes'>attribute types</a> for more info.")
+				.defaultExpression(new EventValueExpression<>(Attribute.class))
+				.examples(attributes.getAllNames())
```

Comment:
This should be changed to `.usage` rather than `.examples` to ensure the names show up in the "pattern" category and not the "examples" category on the docs